Kickstarter-iOS/Views/Controllers/PledgeExpandableRewardsHeaderViewController.swift

```diff
@@ -94,13 +129,69 @@ final class PledgeExpandableRewardsHeaderViewController: UIViewController {
       .observeForUI()
       .observeValues { [weak self] data in
         self?.dataSource.load(data)
+        self?.tableView.reloadData()
+        self?.tableView.setNeedsLayout()
+
+        self?.updateTableViewContainerHeight(expanded: false, animated: false)
       }
+
+    self.viewModel.outputs.expandRewards
+      .observeForUI()
+      .observeValues { [weak self] expanded in
+        self?.updateTableViewContainerHeight(expanded: expanded, animated: true)
+      }
+  }
+
+  private func updateTableViewContainerHeight(expanded: Bool, animated: Bool) {
+    DispatchQueue.main.async {
+      guard animated else {
+        self.tableViewContainerHeightConstraint?.constant = self.collapsedHeight()
+        self.expandButton.transform = self.expandButton.transform.rotated(by: .pi)
+        return
+      }
+
+      if expanded {
+        self.tableViewContainerHeightConstraint?.constant = self.expandedHeight()
+      } else {
+        self.tableViewContainerHeightConstraint?.constant = self.collapsedHeight()
+      }
+
+      UIView.animate(
+        withDuration: 0.33,
```

Comment:
Do we have standardized animations? Unifying our types of animations with consistent durations might be worth it in the long run

Reply:
Not really, I just tweaked the timings here to what felt right in terms of duration and springiness. I'll review this with @colleenmacd to be sure we got it right 👍